internal/connector/pingone/platform/resources/pingone_gateway_role_assignment.go

```diff
@@ -59,13 +59,13 @@ func (r *PingoneGatewayRoleAssignmentResource) ExportAll() (*[]connector.ImportB
 						return nil, err
 					}
 
-					for roleAssignmentIndex, roleAssignment := range gatewayRoleAssignmentsEmbedded.GetRoleAssignments() {
+					for _, roleAssignment := range gatewayRoleAssignmentsEmbedded.GetRoleAssignments() {
 						roleAssignmentId, roleAssignmentIdOk := roleAssignment.GetIdOk()
-
-						if roleAssignmentIdOk {
+						roleAssignmentRoleName, roleAssignmentRoleOk := roleAssignment.GetRoleOk()
```

Comment:
Does Gateway have the idea of AccessControl or does the application role assignment have the GetRoleOk() function?

Reply:
Nevermind, I see the approleassignment moved to using GetRoleOk as well